### PR TITLE
N33: Add a link to a new book

### PR DESCRIPTION
**Pull request by paytonrules**
Add a link and description to a new book about Rust gamedev.

https://github.com/rust-gamedev/rust-gamedev.github.io/issues/1010

### CODE REVIEW

File: content/news/033/index.md [hunk 1]
```diff
@@ -72,6 +72,33 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+adding sprites, sounds, and user input. As you advance, you'll discover how to
+implement a procedurally generated world. Finally, you'll learn how to keep your
+Rust code clean and organized so you can continue to implement new features and
+deploy your app on the web.
+
+_Discussions:
+[twitter](https://www.twitter.com/paytonrules),
+[discord](https://discord.gg/EjAunSbpdV)_
+
+[Game Itself]: https://rust-games-webassembly.netlify.app
```
Comment (ozkriff) [inline content/news/033/index.md:100]: ?

File: content/news/033/index.md [hunk 2]
```diff
@@ -72,6 +72,33 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Learning Material Updates
 
+### Game Development with Rust and WebAssembly
```
Comment (ozkriff) [inline content/news/033/index.md:75]: Could you please add a ink to the book here?

File: content/news/033/index.md [hunk 3]
```diff
@@ -72,6 +72,33 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+
+![Game Development with Rust and WebAssembly Book Cover](https://user-images.githubusercontent.com/21752/166943991-09f449bc-27f1-431e-ad32-a3ab8ccbcbbc.png)
```
Comment (ozkriff) [inline content/news/033/index.md:77]: Could you please commit the image to the repo too? Note that it should be under 300kb.

**Review by ozkriff — CHANGES_REQUESTED** [review]
Looks nice in general but a couple of style tweaks are needed

File: content/news/033/index.md (hunk 2 above)
Comment (paytonrules) [reply content/news/033/index.md:75]: 👍

File: content/news/033/index.md (hunk 1 above)
Comment (paytonrules) [reply content/news/033/index.md:100]: That was meant to be a link - fixed.

File: content/news/033/index.md (hunk 3 above)
Comment (paytonrules) [reply content/news/033/index.md:77]: 👍

**Comment by paytonrules** [comment]
Fixed I think

**Review by 17cupsofcoffee — APPROVED** [review]
Made a few tweaks to the styles/wording, think it's ready to go now :+1:

**Review by ozkriff — APPROVED** [review]
:+1: 

though i'd move the Discord&Twitter links to the beginning of the section as the "discussions" list is meant for specific discussions that have already happened

**Review by AngelOnFira — APPROVED** [review]

**Pull request merged**